src/App.tsx

```diff
@@ -0,0 +1,54 @@
+import { Routes, Route } from "react-router-dom";
+import Main from "../src/app/Main";
```

Comment:
why you are importing using relative path with `../src`? Please find another way. Just use `./`